### ISSUE TITLE
migrate sql blocks current Django LTS (2.10)

### ISSUE THREAD

**Issue by dimitrismistriotis** [klichukb/django-migrate-sql#11]
I guess project is no longer actively maintained, in any case had issue with the following:

Added it to stackoverflow initially: <https://stackoverflow.com/questions/59428173/django-2-2-make-migrations-fails-with-cryptic-attributeerror-node-object-has>

Running migrations on Django 2.10 gives the following output:
```
Traceback (most recent call last):
  File "manage.py", line 16, in <module>
    execute_from_command_line(sys.argv)
  File "/path/to/virtualenv/lib/python3.7/site-packages/django/core/management/__init__.py", line 381, in execute_from_command_line
    utility.execute()
  File "/path/to/virtualenv/lib/python3.7/site-packages/django/core/management/__init__.py", line 375, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/path/to/virtualenv/lib/python3.7/site-packages/django/core/management/base.py", line 323, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/path/to/virtualenv/lib/python3.7/site-packages/django/core/management/base.py", line 364, in execute
    output = self.handle(*args, **options)
  File "/path/to/virtualenv/lib/python3.7/site-packages/migrate_sql/management/commands/makemigrations.py", line 119, in handle
    migration_name=self.migration_name,
  File "/path/to/virtualenv/lib/python3.7/site-packages/django/db/migrations/autodetector.py", line 43, in changes
    changes = self._detect_changes(convert_apps, graph)
  File "/path/to/virtualenv/lib/python3.7/site-packages/django/db/migrations/autodetector.py", line 186, in _detect_changes
    self.generate_altered_fields()
  File "/path/to/virtualenv/lib/python3.7/site-packages/migrate_sql/autodetector.py", line 255, in generate_altered_fields
    self.generate_sql_changes()
  File "/path/to/virtualenv/lib/python3.7/site-packages/migrate_sql/autodetector.py", line 229, in generate_sql_changes
    keys = self.assemble_changes(new_keys, changed_keys, self.to_sql_graph)
  File "/path/to/virtualenv/lib/python3.7/site-packages/migrate_sql/autodetector.py", line 95, in assemble_changes
    ancs = node.ancestors()[:-1]
AttributeError: 'Node' object has no attribute 'ancestors'
```

Most important last: `AttributeError: 'Node' object has no attribute 'ancestors'`. I guess Django added something that was optional and is not supported any more.

**Comment by browniebroke** [comment]
In case you're interested, there is a fork that should be compatible with Django 2.1 - 3.0: 

https://pypi.org/project/django-migrate-sql-deux/

**Comment by dimitrismistriotis** [comment]
@browniebroke Seems that you are very active on your commit history and will keep you in mind. For the time being stopped using it. I think it has to do with small projects - I have dropped support for one of my own - with me being scared of having yours not being maintained after some time for reasons this one got in the same situation (which is 100% OK or I have also done it).

Closing issue by recommending to use the fork. THANKS.
